CODEOWNERS

```diff
@@ -457,6 +457,7 @@
 /samples/bluetooth/mesh/                  @nrfconnect/ncs-paladin
 /samples/bluetooth/multiple_adv_sets/     @nrfconnect/ncs-si-muffin
 /samples/bluetooth/nrf_dm/                @nrfconnect/ncs-si-muffin
+/samples/bluetooth/path_loss_monitoring/  @nrfconnect/ncs-dragoon
```

Comment:
Please add another entry for the RST after /samples/bluetooth/nrf_dm/*.rst           @nrfconnect/ncs-si-muffin-doc: 
/samples/bluetooth/path_loss_monitoring/*.rst  @nrfconnect/ncs-dragoon-doc
Comment